lab_1_keywords_tfidf/main.py

```diff
@@ -33,8 +47,19 @@ def remove_stop_words(tokens: list[str], stop_words: list[str]) -> Optional[list
 
     In case of corrupt input arguments, None is returned
     """
-    pass
-
+    if isinstance(tokens, list) and isinstance(stop_words, list):
```

Comment:
revert the condition and return None immediately

Reply:
Done